docs/csharp/programming-guide/main-and-command-args/main-return-values.md

```diff
@@ -17,7 +17,12 @@ It can also return an `int`:
 
  [!code-csharp[csProgGuideMain#13](~/samples/snippets/csharp/VS_Snippets_VBCSharp/csProgGuideMain/CS/Class3.cs#13)]
 
-If the return value from `Main` is not used, returning `void` allows for slightly simpler code. However, returning an integer enables the program to communicate status information to other programs or scripts that invoke the executable file. The return value from `Main` is treated as the exit code for the process. If `void` is returned from `Main`, the exit code will be implicitly `0`. The following example shows how the return value from `Main` can be accessed.
+If the return value from `Main` is not used, returning `void` allows for slightly simpler code. However, returning an integer enables the program to communicate status information to other programs or scripts that invoke the executable file.
```

Comment:
Not related to your PR, but this article should include that `Main` can return `Task` or `Task<int>`.

If we don't add it now, we should make an issue for that.

Reply:
Adding it now.